app/models/learning_hour.rb

```diff
@@ -52,7 +52,6 @@ def user_org_learning_topic_enable?
 #  id                     :bigint           not null, primary key
 #  duration_hours         :integer          not null
 #  duration_minutes       :integer          not null
-#  learning_type          :integer          default(5)
```

Comment:
why is this diff here?

Reply:
I think it got added by mistake while migrating database locally. Will remove it.